src/main/java/com/daitangroup/App.java

```diff
@@ -0,0 +1,24 @@
+package com.daitangroup;
+
+import org.apache.log4j.BasicConfigurator;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
+    {
+
+        //log4j configuration
+        BasicConfigurator.configure();
```

Comment:
I think it's not really required... maybe you should use another log4j "wrapper" that does it automatically.